utils/lib/rr_graph/channel.py

```diff
@@ -1053,7 +1053,7 @@ def test_pad():
     >>> docprint(test_pad())
     ['| 0  | 1  | 2  | 3  | 4  | 5  |',
      '       AA>---------------->AA ',
-     '       BB>------>BB   XX   XX ',
```

Comment:
I think this is the wrong change, but thought it'd be easier to ask in review than look closer last week.

Reply:
@mithro thoughts on this?

Reply:
Does the test now pass?

Reply:
It looks like the `g.fill_empty(0, name='XX')` isn't working?

Reply:
Fixed properly now. Using `yield` with calls that modify state is a bad combination.